std/experimental/allocator/package.d

```diff
@@ -566,15 +566,18 @@ nothrow @safe @nogc @property void theAllocator(IAllocator a)
     // Install a new allocator that is faster for 128-byte allocations.
     import std.experimental.allocator.building_blocks.free_list : FreeList;
     import std.experimental.allocator.gc_allocator : GCAllocator;
-    auto oldAllocator = theAllocator;
-    scope(exit) theAllocator = oldAllocator;
-    theAllocator = allocatorObject(FreeList!(GCAllocator, 128)());
+    auto oldAllocator = threadAllocator;
+    scope(exit) threadAllocator = oldAllocator;
+    threadAllocator = allocatorObject(FreeList!(GCAllocator, 128)());
     // Use the now changed allocator to allocate an array
-    const ubyte[] arr = theAllocator.makeArray!ubyte(128);
+    const ubyte[] arr = threadAllocator.makeArray!ubyte(128);
     assert(arr.ptr);
     //...
 }
 
+deprecated("Deprecated alias for threadAllocator. This will be removed in a later compiler release.")
+alias theAllocator = threadAllocator;
```

Comment:
I'd always opt for doing at least a short deprecation phase unless that means considerable work. As soon as there is a potentially large number of users, this can save a lot of headaches.

I'll take your snippet and remove it if deprecations actually shouldn't be used here.